### PR TITLE
Fixes dynamic web content

### PR DESCRIPTION
**Pull request by alanhartless**
**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) |  https://github.com/mautic/mautic/issues/8510
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
A dangling line of code with a call to removed deprecated method caused dynamic content to fail rendering. 

Also, fingerprinting had a built in delay that allowed DWC to work. But with the removal of fingerprinting and thus the delay, DWC did not register its pre event callback before the first event was delivered. So DWC was never executed. This is fixed by simply ensuring that the tracking pixel code is the last to load in the mtc.js file giving the chance for all other code to setup before the first event is delivered. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create a dynamic content item (can turn off campaign based and add a filter for this one)
2. Create a landing page and add a Dynamic content slot; use the slot name used in the item created above
3. Browse to the landing page's URL
4. Note the error page

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Repeat and the content should show
3. Test by embedded the DWC into a 3rd party page per Mautic's docs https://www.mautic.org/docs/en/dwc/index.html. The DWC should show.

### CODE REVIEW

File: app/bundles/PageBundle/EventListener/BuildJsSubscriber.php [hunk 1]
```diff
@@ -102,7 +103,7 @@ public function onBuildJs(BuildJsEvent $event)
             
             return;
         }
-
+console.log(m.preEventDeliveryQueue.length, m.beforeFirstDeliveryMade);
```
Comment (escopecz) [inline app/bundles/PageBundle/EventListener/BuildJsSubscriber.php]: Forgotten debug?

Comment (alanhartless) [reply app/bundles/PageBundle/EventListener/BuildJsSubscriber.php]: doh, yes

Comment (alanhartless) [reply app/bundles/PageBundle/EventListener/BuildJsSubscriber.php]: Fixed

**Comment by oltmanns-leuchtfeuer** [comment]
Whilst testing I tried to load the PR in the Mautibox, but I got the following error:

Screenshot at Mar 26 11-08-55

**Comment by npracht** [comment]
Did you try with http://m3.mautibox.com/ ?

**Comment by oltmanns-leuchtfeuer** [comment]
> Did you try with http://m3.mautibox.com/ ?

Weeeell, that was the reason. :D

**Comment by npracht** [comment]
@escopecz can you check Alan’s update ?

**Pull request merged**